shared/chat/inbox/container/filtered.js

```diff
@@ -8,74 +8,111 @@ const score = (lcFilter: string, lcYou: string, names: Array<string>): number =>
     return lcYou.indexOf(lcFilter) !== -1 ? 1 : 0
   }
 
-  const namesMinusYou = names.filter(n => n !== lcYou)
-  return (
-    namesMinusYou.reduce((total, p) => {
-      if (p === lcFilter) {
-        return total + 100 // exact match
+  let namesMinusYou = names.filter(n => n !== lcYou)
+  // special case, comma search
+  const filters = lcFilter.split(',').filter(Boolean)
+  let filter
+  if (filters.length > 1) {
+    const mustExist = filters.slice(0, -1)
+    const partial = filters.slice(-1)
+    // any names between commas must be exact matches
```

Comment:
"between commas" sounds like "all but the first and last", but it looks like you mean "all but the last". change to "all names except the last one must be exact matches"?